src/config/config.rs

```diff
@@ -7,6 +7,7 @@ use async_graphql::parser::types::ServiceDocument;
 use derive_setters::Setters;
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
+use tailcall_macros::MergeRight;
```

Comment:
```suggestion
use tailcall::macros::MergeRight;
```

Reply:
Actually `use crate::macros::MergeRight` would be the better. What do you think?